packages/Algernon.package/ALGCache.class/instance/getAt..st

```diff
@@ -1,6 +1,6 @@
 caching
 getAt: aCacheKey
```

Comment:
I think we should overwrite the directory methods directly. Having both " at:" and "getAt:" at the same time is not really pretty and might confuse later developers.

Reply:
Did that.